dashboard/src/components/Config/AppRepoList/AppRepoAddDockerCreds.tsx

```diff
@@ -30,9 +64,148 @@ export class AppRepoAddDockerCreds extends React.Component<IAppRepoFormProps, {}
         ) : (
           <span className="margin-b-small">No existing credentials found.</span>
         )}
+        {this.state.showSecretSubForm && (
+          <div className="secondary-input margin-t-big">
+            <div className="row">
+              <div className="col-1 margin-t-normal">
+                <label htmlFor="kubeapps-docker-cred-secret-name">Secret Name</label>
+              </div>
+              <div className="col-11">
+                <input
+                  id="kubeapps-docker-cred-secret-name"
+                  value={this.state.secretName}
+                  onChange={this.handleSecretNameChange}
+                  placeholder="Secret"
+                  required={true}
+                />
+              </div>
+            </div>
+            <div className="row">
+              <div className="col-1 margin-t-normal">
+                <label htmlFor="kubeapps-docker-cred-server">Server</label>
+              </div>
+              <div className="col-11">
+                <input
+                  id="kubeapps-docker-cred-server"
+                  value={this.state.server}
+                  onChange={this.handleServerChange}
+                  placeholder="docker.io"
```

Comment:
Let's use the full correct placeholder for docker.io? https://index.docker.io/v1